src/main/java/kitchenpos/products/tobe/domain/DisplayedName.java

```diff
@@ -0,0 +1,35 @@
+package kitchenpos.products.tobe.domain;
+
+import javax.persistence.Embeddable;
+import java.util.Objects;
+
+@Embeddable
+public class DisplayedName {
+    private String name;
+    protected DisplayedName() {
+
+    }
+
+    public DisplayedName(String name) {
+        this.name = name;
+    }
+
+
+
+    public String getValue() {
```

Comment:
```suggestion
@Embeddable
public class DisplayedName {
    private String name;
    
    protected DisplayedName() {

    }

    public DisplayedName(String name) {
        this.name = name;
    }

    public String getValue() {
```

비속어에 대한 책임을 `DisplayedName` 이 가지면 좋을 것 같은데요. 이전에 구조는 잘 잡아 주신 것 같은데 `DisplayedName` 에서 제외한 이유가 따로 있을까요? 🤔 지금 DisplayedName은 비속어도 받을 수 있는 구조라서요. 다른 사람들이 DisplayedName만 보고 비속어가 사용될 수 없음을 알 수 있을까요?

Reply:
고수준 모듈과 저수준 모듈의 개념을 모르고 수정하다 보니 이렇게 되었네요. 😭 
고수준 모듈이 좀더 추상적인, 또는 로직을 가지고 있고, 저수준 모듈이 VO나 DTO같은 객체라고 생각해 설계를 고쳤는데요;;

다시 보니, 고수준과 저수준이라는 건, `Presentation Layer > Application Layer > Domain Layer > Infrastructure Layer`로 가는 의존 흐름에 따른 거더군요. 
계층형 아키텍처에서 위쪽에 있을수록 고수준, 아래쪽에 있으면 저수준 모듈이더라구요!
PurgomalumProfanityClient는 인프라 계층으로 저수준 모듈이고, 이것을 추상화한 ProfanityClient는 도메인 계층으로 고수준 모듈로 볼 수 있겠단 생각이 들었어요.


제가 이해한게 맞겠죠...? 🤔 리뷰 주신 데로 수정해보겠습니다. 😄